packages/snap/snap.manifest.json

```diff
@@ -21,47 +21,51 @@
     "snap_dialog": {},
     "snap_getBip32Entropy": [
       {
-        "path": ["m", "44'", "0'"],
+        "path": [
+          "m",
+          "44'",
+          "0'"
+        ],
         "curve": "secp256k1"
       },
       {
-        "path": ["m", "44'", "2'"],
+        "path": [
+          "m",
+          "44'",
+          "2'"
+        ],
         "curve": "secp256k1"
       },
       {
-        "path": ["m", "44'", "3'"],
+        "path": [
+          "m",
+          "44'",
+          "3'"
+        ],
         "curve": "secp256k1"
       },
       {
-        "path": ["m", "44'", "60'"],
+        "path": [
+          "m",
+          "44'",
+          "118'"
+        ],
         "curve": "secp256k1"
       },
       {
-        "path": ["m", "44'", "118'"],
+        "path": [
+          "m",
+          "44'",
+          "145'"
+        ],
         "curve": "secp256k1"
       },
       {
-        "path": ["m", "44'", "145'"],
-        "curve": "secp256k1"
-      },
-      {
-        "path": ["m", "44'", "714'"],
-        "curve": "secp256k1"
-      },
-      {
-        "path": ["m", "44'", "931'"],
-        "curve": "secp256k1"
-      },
```

Comment:
145 and 931 are BCH and RUNE, both present on the right, simply linted - 714 is BNB (Beacon Chain) which we don't explicitly support